script.js

```diff
@@ -1,12 +1,35 @@
+const seeProjectButtons = document.querySelectorAll('.see-project-button');
+
 document.querySelector('.menu-button').addEventListener('click', () => {
   document.querySelector('.mobile-menu').classList.toggle('appear');
   document.querySelector('.menu-button').classList.toggle('cross-button');
 });
+const navItems = document.querySelectorAll('.mobile-menu li a');
 
-for (let i = 0; i < document.querySelectorAll('.mobile-menu li a').length; i += 1) {
-  const selectedElement = document.querySelectorAll('.mobile-menu li a')[i];
-  selectedElement.addEventListener('click', () => {
+// for (let i = 0; i < document.querySelectorAll('.mobile-menu li a').length; i += 1) {
+//   const selectedElement = document.querySelectorAll('.mobile-menu li a')[i];
+//   selectedElement.addEventListener('click', () => {
+//     document.querySelector('.mobile-menu').classList.toggle('appear');
+//     document.querySelector('.menu-button').classList.toggle('cross-button');
+//   });
+// }
```

Comment:
- [ ] [OPTIONAL] Let's remove the commented line of code in order to have a clean ode base.